holoviews/plotting/renderer.py

```diff
@@ -484,7 +492,7 @@ def open_and_read(path, f):
         if extras:
             dependencies.update(cls.extra_dependencies)
         for backend in backends:
-            dependencies['backend'] = Store.renderers[backend].backend_dependencies
+            dependencies[backend] = Store.renderers[backend].backend_dependencies
```

Comment:
Separate fix that snuck in because I've had it on my working tree for a week. Happy to make a new PR, doesn't seem worthwhile though.

Reply:
What was going on here before?! What was the broken behavior and what is the new behavior?

Reply:
Nothing was actually broken in any usual scenario, since you'd usually only request to embed the JS/CSS dependencies for one backend in an HTML file. In theory though you could do something crazy like embed a plotly plot in a bokeh Div and try to export that to HTML.